blocks/library/heading/index.js

```diff
@@ -41,9 +41,9 @@ registerBlockType( 'core/heading', {
 				transform: ( { content, ...attrs } ) => {
 					const isMultiParagraph = Array.isArray( content ) && isObject( content[ 0 ] ) && content[ 0 ].type === 'p';
 					if ( isMultiParagraph ) {
-						const headingContent = isString( content[ 0 ] )
-							? content[ 0 ]
-							: content[ 0 ].props.children;
+						const headingContent = isObject( content[ 0 ] ) && content[ 0 ].type === 'p'
```

Comment:
I guess it doesn't matter much here, but not sure why we should change it, the string check being faster.